### DISCUSSION TITLE
Add an option to hide the extension in the top panel

### DISCUSSION THREAD

**Discussion by Netherquark** [nlpsuge/gnome-shell-extension-another-window-session-manager#85] [Ideas]
For my usecase, I'm only going to use the save session on logout, and restore session automatically settings. I don't need to have the extension in my top panel 24x7, because though its running 24x7, it'll do everything it needs to automatically.

Would it be possible to add this option similar to how the "Emoji Selector" extension has it?

For reference:

![image](https://github.com/nlpsuge/gnome-shell-extension-another-window-session-manager/assets/65618735/903182d6-a909-411f-ad65-c0c372145b14)

**Comment by nlpsuge** [comment]
Sure. I have created an issue (https://github.com/nlpsuge/gnome-shell-extension-another-window-session-manager/issues/86) for it.
